core/scheduler/service/events_service.go

```diff
@@ -146,6 +162,12 @@ func (e *EventsService) Push(ctx context.Context, event *scheduler.Event) error
 		return nil
 	}
 
+	if e.IsBackFill(event) {
```

Comment:
can we just depends on `Relay` function to trigger the sla_miss? instead of `Push`? so you only need to whitelist optimus-job-sla-miss template on alert rules (see optimus.go). you don't need to recreate `IsBackfill` function like this

Reply:
ideally we want to deprecate the direct integration flow so coupling this with the relay function might be unnecessary effort